task09.rb

```diff
@@ -0,0 +1,20 @@
+# In a string move the letters to the next letter in the alphabet, e.g. "quiz" should become "rvja".
+
+def task09(str)
+  raise ArgumentError, "Not a valid format. Need a string" if str.class != String
+  raise ArgumentError, "String is empty. This is not valid" if str.empty?
+  raise ArgumentError, "Only letters are allowed" if str.scan(/^[a-zA-Z]+$/).empty?
```

Comment:
@MaraEfimova `" j"`, `" j "` and `"j "` are valid inputs, but the method throws an error. And with `"     "` the error message is a bit misleading...